### PR TITLE
Remove duplicate created-at column

### PR DESCRIPTION
**Pull request by mociarain**
Fixs R report bug where the created_at was duplicated

### CODE REVIEW

**Review by eliasbenussi — APPROVED** [review]

File: R/reports.R [hunk 1]
```diff
@@ -82,7 +82,7 @@ extract_report_info <- function(report_object) {
       magrittr::extract(
         report_object$active_version,
         c("version_id", "report_path", "author_id",
-          "notebook_path", "created_at", "status")
+          "notebook_path", "status")
```
Comment (tommilata) [inline R/reports.R:85]: I've got almost no context on this codebase, but was the problem that we were taking both `created_at` from this line (from the active version) and the `created_at` from report entity (a few lines above)?

Just out of curiosity, what is this `extract_report_info` used for and why do we want to pass `created_at` from the report and not from the version?

Comment (mociarain) [reply R/reports.R:85]: You're right about the error.

My loose understanding is that it was causing issues further up in the `purrr::map_df` call on line 101. Tbh I find the R code very hard to follow and made an educated guess based on input from Pascal and the error message.
